src/Tools/SemanticSearch/ReferenceAssemblies/SemanticSearch.ReferenceAssemblies.csproj

```diff
@@ -49,7 +49,7 @@
     </ItemGroup>
   </Target>
   
-  <Target Name="GenerateFilteredReferenceAssemblies" BeforeTargets="CoreCompile" DependsOnTargets="_CalculateFilteredReferenceAssembliesInputsAndOutputs" Inputs="@(_InputFile)" Outputs="@(_OutputFile)">
+  <Target Name="GenerateFilteredReferenceAssemblies" Condition="Exists('$(ArtifactsBinDir)SemanticSearch.BuildTask\$(Configuration)\$(_BuildTaskTfm)\SemanticSearch.BuildTask.dll')" BeforeTargets="CoreCompile" DependsOnTargets="_CalculateFilteredReferenceAssembliesInputsAndOutputs" Inputs="@(_InputFile)" Outputs="@(_OutputFile)">
```

Comment:
Can we store the path to a property and use it here and on line 29 to avoid duplication?

BTW, what are the conditions under which the file does not exist?

Reply:
> BTW, what are the conditions under which the file does not exist?

After Git Clean is one case.